### DISCUSSION TITLE
Pull Request Templates - Should Allow us to also have a template for PR Title

### DISCUSSION THREAD

**Discussion by markalanevans** [community/community#63965] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

in the template .md file

I should be able to specify:

Title:  Feature|Chore|BugFix - INSERT_USER_STORY

like we can in an issue template

**Comment by spenserblack** [comment]
Honestly, I wish PR templates and issue templates behaved the same way for a lot of features. The fact that PR templates still seem to do little more than copy the contents from the file to the PR body, even though issue templates have so many features (and now we have [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)), seems strange to me.

**Reply by markalanevans** [reply]
Agreed. They could basically work the same.

**Reply by jbsmith7741** [reply]
and they should

**Comment by wt-asw** [comment]
I have also tried setting up PR templates for my repos and was confused that they did not behave in the same way as the issue templates. 

Would be good if they did :)

**Comment by jbsmith7741** [comment]
Related to this issue before github code was archived. https://github.com/isaacs/github/issues/1252

**Comment by deleugpn** [comment]
Anybody managed to accomplish this in any way?

**Comment by VBardin08** [comment]
I'd like to see the pull requests template feature with more valuable things like:
- creating a form (similar to issue templates)
- having the ability to define a template for the PR header as well

**Comment by ryancouto** [comment]
+1 would love to see title templates
